src/Net/HostnameToIpResolver.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RapidCore.Net
```

Comment:
I would suggest `RapidCore.Network`. I have no real reason except that I think it sounds better :)